packages/ap-frontend/alliance_platform/frontend/alliance_ui/misc.py

```diff
@@ -38,6 +39,7 @@ def render_html(self, context):
 
 
 def raw_html(parser: template.base.Parser, token: template.base.Token):
+    """Render a raw HTML component - can be used to pass HTML children to React components."""
```

Comment:
This is probably not required anymore as you can just render HTML directly as children now, and as it depends on "components/RawHtmlWrapper" existing I'm happy to just remove it. If it's actually needed it can be achieved using the component tag directly.